requirements_big_mode.txt

```diff
@@ -6,3 +6,4 @@ asgi-redis>=1.3,<1.4
 django-redis>=4.7.0,<4.8
 django-redis-sessions>=0.5.6,<0.6
 psycopg2>=2.7,<2.8
+txredisapi==1.4.4
```

Comment:
Please test (now or later) Windows Portable and install steps on fresh Ubuntu 16.04 so that this new package does not bring new headache on release day. :-)

Reply:
"pip install txredisapi" (v1.4.4) successfully tested on Windows (Portable) and Linux (arch, ubuntu 16.10).